constraint_builder.h

```diff
@@ -66,14 +101,30 @@ class ConstraintBuilder {
         return variableInRange(nthVelocity(i), l, u);
     }
 
+    ConstraintBuilder &accelerationInRange(size_t i,
+                                           constraint_t l = NEG_INF<N_DIM>,
+                                           constraint_t u = POS_INF<N_DIM>) {
+        assert(i < waypoints);
+        size_t baseA = nthAcceleration(i);
+        size_t baseV = nthVelocity(i);
+        for(int j = 0; j < N_DIM; j++) {
+            addConstraint(userConstraintOffset + baseA + j,
+                          {
+                            factor_t{baseV + j + N_DIM,  1},
```

Comment:
Please change it to sth like baseNextV + j, where baseNextV = nthVelocity(i + 1)